recipes/so3/all/conanfile.py

```diff
@@ -0,0 +1,62 @@
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+from glob import glob
+import os
+
+
+class So3Conan(ConanFile):
+    name = "so3"
```

Comment:
```suggestion
    name = "astro-informatics-so3"
```

I would rename it. The project SO3 provided by Astro Informatics is not so popular, only 3 github stars and there are other similar projects with same name.

Reply:
Fair enough. We'll go with so3-transform, if that's allright. It will have to wait a few days though.

Reply:
@uilianries  unfortunately, the name so3 might be too generic (e.g. it stands for sulfur trioxide and few other things)

Reply:
https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-is-the-policy-on-recipe-name-collisions

https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#should-reference-names-use---or-_

> For libraries with a too generic name, like variant, the name of the organization can be used as prefix separated by a -, like mpark-variant

Our "rule" is the author/organization if you dont mind following the suggestion above